src/main/kotlin/no/brreg/informasjonsforvaltning/organizationcatalogue/mapping/Mappers.kt

```diff
@@ -58,6 +59,33 @@ fun OrganizationDB.updateValues(org: Organization): OrganizationDB =
         allowDelegatedRegistration = org.allowDelegatedRegistration ?: allowDelegatedRegistration
     }
 
+fun OrganizationDB.updateWithEnhetsregisteretValues(org: EnhetsregisteretOrganization): OrganizationDB =
+    apply {
+        name = org.navn ?: name
+        orgType = org.organisasjonsform?.kode
+        orgPath = org.orgPath
+        subOrganizationOf = org.overordnetEnhet
+        municipalityNumber = org.forretningsadresse?.kommunenummer ?: org.postadresse?.kommunenummer
+        issued = org.registreringsdatoEnhetsregisteret?.let { LocalDate.parse(it) }
+        industryCode = org.naeringskode1?.kode
+        sectorCode = org.institusjonellSektorkode?.kode
+        prefLabel = if (prefLabel.isNullOrEmpty()) org.prefLabelFromName() else prefLabel
+    }
+
+private fun EnhetsregisteretOrganization.prefLabelFromName(): PrefLabel =
+    PrefLabel().apply {
+        nb = navn?.toLowerCase()?.capitalize()
```

Comment:
Dette er vel ikke alltid riktig, men er godt nok for fleste tilfeller

Reply:
Er omtrent det Joachim sa når vi snakket om det på fredag